src/sentry/static/sentry/app/views/organizationDashboard/emptyState.jsx

```diff
@@ -3,36 +3,78 @@ import PropTypes from 'prop-types';
 import {Flex} from 'grid-emotion';
 import styled from 'react-emotion';
 
-import {t} from '../../locale';
+import {t} from 'app/locale';
+import Button from 'app/components/buttons/button';
+import Tooltip from 'app/components/tooltip';
+import SentryTypes from 'app/proptypes';
 import img from '../../../images/dashboard/hair-on-fire.svg';
-import Button from '../../components/buttons/button';
 
 export default class EmptyState extends React.Component {
   static propTypes = {
-    orgId: PropTypes.string,
+    organization: SentryTypes.Organization,
+    projects: PropTypes.arrayOf(SentryTypes.Project),
   };
 
   render() {
-    const {orgId} = this.props;
+    const {organization, projects} = this.props;
+    const orgId = organization.slug;
+    const canCreateProject = organization.access.includes('project:write');
+    const canJoinTeam = organization.access.includes('team:read');
+    const hasProjects = !!projects.length;
+
     return (
       <Flex flex="1" align="center" justify="center">
         <Wrapper>
           <img src={img} height={350} alt="Nothing to see" />
           <Content direction="column" justify="center">
             <h2>{t('Remain calm.')}</h2>
-            <p>{t("Sentry's got you covered.")}</p>
-            <div>
-              <Button priority="primary" to={`/organizations/${orgId}/projects/new/`}>
-                {t('Create project')}
-              </Button>
-            </div>
+            <p>{t("Sentry's got you covered. To get started:")}</p>
+            <Flex align="center">
+              <CallToAction>
+                <Tooltip
+                  disabled={canJoinTeam}
+                  title={t('You do not have permission to join a team.')}
+                >
+                  <Button
+                    disabled={!canJoinTeam}
+                    priority="primary"
+                    to={`/settings/${orgId}/teams/`}
+                  >
+                    {t('Join a Team')}
+                  </Button>
+                </Tooltip>
+              </CallToAction>
+
+              {!hasProjects && (
```

Comment:
Let's remove this logic and always show the button, since it's now deemphasized.